src/app/(post-login)/submission/program/[shortName]/clinical-data/common.tsx

```diff
@@ -209,8 +211,9 @@ export const hasClinicalErrors = (
 			),
 	).length > 0;
 
-export const emptyClinicalDataResponse: ClinicalEntityQueryResponse = {
+export const emptyClinicalDataResponse: SubmittedDataSideMenuQuery = {
 	clinicalData: {
+		programShortName: '',
```

Comment:
not ideal but works ok as a type-fix - this is just an empty object meant as a default value to display nothing - "emtptyClinicalDataResponse"

the root cause is the mismatch between TS types and GQL generated types.